sender.py

```diff
@@ -0,0 +1,29 @@
+import paho.mqtt.client as mqtt
+
+from generator import Generator, object1
+
+
+class Sender(Generator):
```

Comment:
Sender should not extend Generator, Generators purpose is to create random numbers, Senders purpose is to publish whatever is passed to MQTT broker

Reply:
but how sender will get numbers from generator without extending?
or idk smth?